.github/blunderbuss.yml

```diff
@@ -1,4 +1,17 @@
+# Blunderbuss config
+#
+# This file controls who is assigned for pull requests and issues.
+# Note: This file is autogenerated. To make changes to the assignee
+# team, please update `codeowner_team` in `.repo-metadata.json`.
 assign_issues:
-  - googleapis/api-logging-reviewers
+  - googleapis/api-logging
+
+assign_issues_by:
+  - labels:
+      - "samples"
+    to:
+      - googleapis/python-samples-reviewers
+      - googleapis/api-logging
+
 assign_prs:
-  - googleapis/api-logging-reviewers
+  - googleapis/api-logging
```

Comment:
@gkevinzheng Can we update the `codeowner_team` to `googleapis/api-logging-reviewers` instead of `googleapis/api-logging` in `.repo-metadata.json`?

https://github.com/googleapis/python-error-reporting/blob/08b9d9f641dc7e02d801066624796e89895836a1/.repo-metadata.json#L14

Reply:
@parthea In `python-logging`, we have 

```
    "codeowner_team": "@googleapis/api-logging @googleapis/api-logging-partners",
```

for that line, so I think I'll make it that for more consistency.